ci/requirements-3.6.run

```diff
@@ -14,7 +14,7 @@ html5lib
 jinja2
 sqlalchemy
 pymysql
-feather-format
+# feather-format (not available on defaults ATM)
```

Comment:
revert the change
you shouldn't be changing this file 
it's for the travis build